cfme/fixtures/v2v_fixtures.py

```diff
@@ -195,18 +256,50 @@ def set_conversion_instance_for_osp(appliance, osp_provider, transformation_meth
     except KeyError:
         pytest.skip("No conversion instance on provider.")
 
-    for instance in conversion_instances:
-        set_conv_host = appliance.ssh_client.run_rails_command(
-            "'r = Vm.find_by(name:{vm});\
-        c_host = ConversionHost.create(name:r.name, resource: r);\
-        c_host.{method}_transport_supported = true;\
-        c_host.save'".format(
-                vm=json.dumps(instance),
-                method=transformation_method.lower(),
+    if ui:
```

Comment:
Conversion host UI modal is almost same for RHV and OSP except TLS usecase, can you use one common function for that and add that back to `set_conversion_instance_for_osp` and `set_conversion_instance_for_rhev` functions, what do you say?